File: mt9x9.f90
! 9x9 multiplication table in Fortran90
! CC0, Wei-Lun Chao <bluebat@member.fsf.org>, 2018.
! gfortran mt9x9.f90 -o mt9x9 ; ./mt9x9

program mt9x9
    implicit none
    integer :: i, j, k
    character(len=24) :: format
    do i = 1, 9, 3
    format = "(3(i1, a, i1, a, i2, a))"
    write(*,format) ((k, 'x', j, '=', k*j, char(9), k = i, i+2), j = 1, 9)
    write(*,*) 
    end do
end program mt9x9

File: mt9x9.js
#!/usr/bin/gjs
/*
9x9 multiplication table in JavaScript
CC0, Wei-Lun Chao <bluebat@member.fsf.org>, 2018.
*/
// ./mt9x9.js || gjs mt9x9.js

for(var i = 1; i <= 9; i += 3) {
    for(var j = 1; j <= 9; j++)
        print(
            [i, i+1, i+2].map(
                k => k+'x'+j+'='+(' '+k*j).slice(-2)
            ).join('\t')
        )
    print()
}
